### PR TITLE
Better operator version

### PR DESCRIPTION
**Pull request by slintes**
With this PR the version info printed by the operator on startup improves (this is on a dirty version, 3 commits after the last tag):
```
$ build/_output/bin/performance-addon-operators
I0429 19:26:39.196108  458318 main.go:47] Operator Version: 1.2.3-4.test-3-g7f0661a
I0429 19:26:39.196166  458318 main.go:48] Git Commit: 7f0661aeca63ceaf52f805b9a649e4253628978a
I0429 19:26:39.196183  458318 main.go:49] Build Date: 2020-04-29T17:26:32+00:00
...
```

**Heads up**: this PR is on top of #196, reviewing the last commit is enough

@fromanirh: this should pick up the right env vars in downstream CI to still display correct upstream version

/hold
until #196 is merged

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -55,6 +55,13 @@ steps:
       repo: quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator
       auto_tag: true
 
+  - name: validate-operator-imgage
+    image: quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator:${DRONE_TAG}
+    commands:
+      - ls /usr/local/bin/ | grep performance-operator
```
Comment (fromanirh) [inline .drone.yml:63]: maybe do directly `ls /usr/local/bin/performance-operator` ? if this fails it's already a bad thing

File: .drone.yml [hunk 2]
```diff
@@ -72,6 +79,13 @@ steps:
       repo: quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator-registry
       auto_tag: true
 
+  - name: validate-registry-image
+    image: quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator-registry:${DRONE_TAG}
+    commands:
+      - sqlite3 /registry/bundles.db "select * from package" | grep "performance-addon-operator|${DRONE_SEMVER_SHORT}"
+      - sqlite3 /registry/bundles.db "select * from channel" | grep "${DRONE_SEMVER_SHORT}|performance-addon-operator|performance-addon-operator.v${DRONE_SEMVER_SHORT}"
+      - sqlite3 /registry/bundles.db "select * from operatorbundle" | grep "\"image\":\"quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator:${DRONE_TAG}\""
```
Comment (fromanirh) [inline .drone.yml:88]: very nice, a bit too low level. Is this supported by the OLM folks?

**Comment by fromanirh** [comment]
looks nice. I'd like to have another look to asses the d/s impact

File: .drone.yml (hunk 1 above)
Comment (slintes) [reply .drone.yml:63]: TIL you can `ls` a file :slightly_smiling_face: I can't remember doing that before, thanks!

File: .drone.yml (hunk 2 above)
Comment (slintes) [reply .drone.yml:88]: answered on #196

**Comment by fromanirh** [comment]
looks fine to me, I have no more comments

**Pull request merged**